mrmustard/physics/triples.py

```diff
@@ -795,6 +795,31 @@ def displacement_map_s_parametrized_Abc(s: int, n_modes: int) -> Union[Matrix, V
     return math.astensor(A), b, c
 
 
+def bargmann_to_wigner_Abc(s: int, n_modes: int) -> Union[Matrix, Vector, Scalar]:
+    r"""
+    The Abc triple of the Bargmann to Wigner/Husimi transformation.
+    """
+
+    On = math.zeros((n_modes, n_modes), dtype=math.complex128)
+    In = math.eye(n_modes, dtype=math.complex128)
+
+    A = (
+        2
+        / (s - 1)
+        * math.block(
+            [
+                [On, -In, In, On],
```

Comment:
isn't this the opposite of the one we have in the stellar decomposition paper? it looks like input and output are swapped, so one of the two must be wrong (or I'm not reading it correctly)

Reply:
ahh I see... ok I'll get back to you on slack